### ISSUE TITLE
Issue loading Parquet files with integer columns narrower than 64 bits

### ISSUE THREAD

**Issue by 21771** [Bears-R-Us/arkouda#2327]
**Describe the bug**
When loading Parquet files that have integer columns narrower than 64 bits, Arkouda corrupts uint32 values whose high bit is set and ignores 16-bit columns entirely. (submitting for colleague)

**To Reproduce**
```python
df_pd = pd.DataFrame({
    "int16":  pd.Series([2**15-1, -2**15], dtype=np.int16),
    "int32":  pd.Series([2**31-1, -2**31], dtype=np.int32),
    "uint16: pd.Series(['2**15-1,  2**15], dtype=np.uint16),
    "uint32: pd.Series(['2**31-1,  2**31], dtype=np.uint32),
})
df_pd.to_arquet("test.parquet")
df_ak = ak.DataFrame(ak.read("test.parquet"))
```

**Expected behavior**
df_ak values to match df_pd values

**Error Message**
No error message

**Is this a Blocking Issue**
Works fine reading hdf5 with smaller column sizes.

**Comment by Ethan-DeBandi99** [comment]
Recording the output before changes put in:

**Pandas Read Output**
```
   int16       int32  uint16      uint32               uint64
0  32767  2147483647   32767  2147483647  9223372036854775807
1 -32768 -2147483648   32768  2147483648  9223372036854775808
```

**Arkouda Read Output**
```
        int32                uint32               uint64
0  2147483647            2147483647  9223372036854775807
1 -2147483648  18446744071562067968  9223372036854775808 (2 rows x 3 columns)
```

**Comment by Ethan-DeBandi99** [comment]
Output following updates:

*Pandas Read Output**
```
   int16       int32  uint16      uint32               uint64
0  32767  2147483647   32767  2147483647  9223372036854775807
1 -32768 -2147483648   32768  2147483648  9223372036854775808
```

**Arkouda Read Output**
```
   int16       int32  uint16      uint32               uint64
0  32767  2147483647   32767  2147483647  9223372036854775807
1 -32768 -2147483648   32768  2147483648  9223372036854775808 (2 rows x 5 columns)

```

**Comment by 21771** [comment]
Thanks

On Monday, April 3, 2023, pierce ***@***.***> wrote:
> Closed #2327 as completed via 1153448.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.<
https://ci5.googleusercontent.com/proxy/2XhDmwQtZWFz01h-QS3fTlRp8RoBJSaZq4b4sZ9cVH16JVD8QXtkF5GsDL0zMF2JktndDTPQsAATMfOqoXu6n-9ktO8xd3hikQXXWHBL8TO-EfJwKmwCK2d-Yvoeynf9kbTY6O8PU7cNis5hEqAcPmfGelQDIJGr8C6DFtqxlsg53A6FCt3i0n31XNDRGqmGVSMWdMCsIBmkA7_rQ8oPhPI20XNm2ATkcntEf_s1U5pkxsr2aNYDXtJbQvD4moerpY5379Tj4iTVJyyN=s0-d-e1-ft#https://github.com/notifications/beacon/AYBTBA6PY5EOAZCFTDOJZJDW7M42XA5CNFSM6AAAAAAWN23PAOWGG33NNVSW45C7OR4XAZNWJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XKUY3PNVWWK3TUL5UWJTYAAAAAEE4F2HJQ.gif>Message
ID: ***@***.***>
